enarx-mirrorlabels

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: Apache-2.0
+
+import enarxbot
+
+TO_RENAME = {
+    'amd sev': 'sev'
+}
+
+github = enarxbot.connect()
+org = github.get_organization('enarx')
+enarx = github.get_repo("enarx/enarx")
+
+ORGANIZATION_LABELS = enarxbot.ORGANIZATION_LABELS
+organization_label_names = {l for l in ORGANIZATION_LABELS.keys()}
+
+for repo in org.get_repos():
+    repo_labels = {l.name: l for l in repo.get_labels()}
+
+    # Rename labels that need to be renamed.
+    for name in TO_RENAME.items():
+        if name[1] not in organization_label_names:
+            print(f"WARNING: {name[1]} is not present in the ORGANIZATION_LABELS list. Skipping rename until it is.")
+            continue
```

Comment:
Moving `TO_RENAME` into the `ORGANIZATION_LABELS` like I suggested makes this error impossible.